modules/ROOT/pages/testing-writing-your-first-munit-test.adoc

```diff
@@ -0,0 +1,201 @@
+= Creating Your Test 
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+To test your module, you must create MUnit tests because MTF is built on top of MUnit. 
+
+This section provides a step-by-step example for testing a module using MUnit. This example is based on the module project generated with the Mule Extensions Archetype in xref:getting-started.adoc#generating-a-project-using-the-maven-archetype-directly[Creating Your First SDK Project].
+
+To test your module, you must:
+
+. <<configure-your-module, Configure your module>>.
+. <<write-your-test, Write your MUnit test>>.
+. <<run-your-test, Run your MUnit test>>.
+
+[[configure-your-module]]
+== Configure Your Module
+
+To configure your module to run your MUnit tests, you must configure your `pom.xml` file:
+
+. Add the following properties:
++
+[source,xml,linenums]
+----
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+   …
+   <properties>
+    <!-- Remove when a new parent version with MTF is available -->
+    <munit.input.directory>src/test/munit</munit.input.directory>
+    <munit.output.directory>${basedir}/target/test-mule/munit</munit.output.directory>
+    <munit.extensions.maven.plugin.version>1.0.0</munit.extensions.maven.plugin.version>
+    <munit.version>2.2.3</munit.version>
+    <mavenResourcesVersion>3.0.2</mavenResourcesVersion>
+    ...
+  </properties>
+  ...
+</project>
+----
+. Add the Maven Resources Plugin:
++
+[source,xml,linenums]
+----
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+   …
+   <build>
+    <plugins>
+      ...
+      <plugin>
+          <artifactId>maven-resources-plugin</artifactId>
+          <version>${mavenResourcesVersion}</version>
+          <executions>
+              <execution>
+                  <id>copy-munit-resources</id>
+                  <phase>process-test-resources</phase>
+                  <goals>
+                      <goal>copy-resources</goal>
+                  </goals>
+                  <configuration>
+                      <outputDirectory>${munit.output.directory}</outputDirectory>
+                      <resources>
+                          <resource>
+                              <directory>${munit.input.directory}</directory>
+                              <filtering>true</filtering>
+                          </resource>
+                      </resources>
+                  </configuration>
+              </execution>
+          </executions>
+      </plugin>
+      ...
+    </plugins>
+  </build>
+  ...
+</project>
+----
+. Add the MUnit Extensions Maven Plugin:
```

Comment:
Capitalization?

Reply:
Depending on how this looks in your build (code samples truncated), you might consider using [%collapsible] to make the procedure (look) shorter.

Reply:
It shows up as collapsible/expandable right now
https://beta.docs.mulesoft.com/beta-mtf/mule-sdk/1.1/testing-writing-your-first-munit-test